Packs/XSOARmirroring/Integrations/XSOARmirroring/XSOARmirroring.py

```diff
@@ -639,8 +623,8 @@ def get_remote_data_command(client: Client, args: dict[str, Any], params: dict[s
             remote_args.last_update = occurred + 1
             # in case new entries created less than a minute after incident creation
```

Comment:
all the code changes are just ruff?

Reply:
that, and my OCD

Reply:
ill remove all changes, it's triggering the build